README.md

```diff
@@ -72,27 +72,148 @@ Pingdom will be integrated with
 - `build.sh` defines the checks and is executed as part of the automated build.
   This script also defines integration IDs.
 - `bin/pingdom` provides basic ability to interact with the Pingdom API.
+- `bin/get-secret` is a utility to extract secrets from AWS Parameter store with the ability
+   to try local secrets first.
 - `templates/*.json` defines check templates in Pingdom API format.
   These checks are processed to substitute template variables and then uploaded
   to Pingdom to create or update checks.
+- `pingdom-checks/*.ping` collection of checks, typically one related group of checks per file.
+  These checks will be processed and updated Pingdom.
+- `integration-ids.conf` contains shared Pingdom integration IDs. 
+   Integration IDs that are not shared can be placed in the `*.ping` health checks directly.
+- `user-ids.conf` contains Pingdom user IDs.
+   User IDs that are not shared can be placed in the `*.ping` health checks directly.
 
 ### Create a new check or update an existing
-Update the `build.sh` to include a new check definition or change an existing check.
-Updates to checks are performed by _name_. Name changes are seen as new checks.
+Create or update `pingdom-checks/*.ping` file to include a new check definition or change an 
+existing check.
+
+> Pro Tip: Create a curl command of the check _first_. Once you done this, translating it
+> to Pingdom is fairly straight forward.
+
+Pingdom checks use a _template_ to create the HTTP request. Many checks follow the same pattern,
```

Comment:
Probably should mention the templates are in the `/templates` directory of the repo.